site/docs/master/overview-plugins.md

```diff
@@ -19,6 +19,9 @@ Anyone can add integrations for any platform to provide additional backup and vo
 
 After you publish your plugin on your own repository, open a PR that adds a link to it under the appropriate list of [supported providers][3] page in our documentation.
 
+You can also add the `velero-plugin` GitHub Topic to your repo, and it be shown under the aggregated list of repositories automatically.
```

Comment:
will it actually be shown automatically?

Reply:
Yeah, the topic URL in the links below is basically a search for all repos mentioning it, so it should be automatic, possibly some slight delays.

Reply:
Ah, I see what you mean. :+1: